Tests/ATTNAPITest.m

```diff
@@ -18,6 +18,7 @@
 #import "ATTNCart.h"
 
 static NSString* const TEST_DOMAIN = @"some-domain";
+static NSString* const TEST_GEO_ADJUSTED_DOMAIN = @"some-domain";
```

Comment:
nit: should we name this domain to `@"some-domain-ca";` or something just to not confuse why we have two of the same variables in this class?

Reply:
😵‍💫 yes, it should be different

Reply:
updated!